src/F2_autoreport.py

```diff
@@ -889,6 +889,27 @@ def create_app(results_json):
     )
     df_capacities.reset_index(drop=True, inplace=True)
 
+    # Data preparation operations for generating the pie charts with Plotly
+
+    # Get the cost matrix from the reuslts JSON file into a pandas DF
+    df_kpis = results_json[KPI][KPI_COST_MATRIX]
+    # List of the needed parameters
+    costs_needed = ["label", ANNUITY_TOTAL, COST_INVESTMENT, COST_OM_TOTAL]
+    # Drop all the irrelevant columns
+    df_kpis = df_kpis[costs_needed]
+    df_kpis = df_kpis.append(df_kpis.sum(numeric_only=True), ignore_index=True)
+    # Add a label for the row holding the sum of each column
+    df_kpis.iloc[17, 0] = "Total"
```

Comment:
`.iloc[17,0]` seems awfully specific.

Maybe this works?

```
    df_kpis["total"] = df_kpis.sum(numeric_only=True)
```

Reply:
Already tried that line and for some reason, it does not work :( 

I will change it to this, which is not specific to the number of assets:
`df_kpis.iloc[-1, 0] = "Total"`

Thanks for pointing this out :)